.github/workflows/package-size.yml

```diff
@@ -0,0 +1,16 @@
+name: Package Size
+
+on: pull_request
+
+jobs:
+  package-size:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: preactjs/compressed-size-action@v1
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
you can add on line 17 `build-script: "build-bundle-size"` and make a new build script just for this that will run bootstrap

Reply:
mmmm good call, updated